core/src/sql/value/value.rs

```diff
@@ -123,7 +124,8 @@ pub enum Value {
 	Regex(Regex),
 	Cast(Box<Cast>),
 	Block(Box<Block>),
-	Range(Box<Range>),
+	#[revision(end = 2, convert_fn = "convert_old_range")]
+	Range(OldRange),
```

Comment:
I guess within a tuple the name of the structure does not matter, only the position is relevant.

Reply:
Yep! That enum variant is no longer in the finally generated enum I think, otherwise rust would have complained 😅 
We can rename that to `OldRange` too if you'd like though? To make it more visible

Reply:
I was referring to the name of the structure inside the variant, not the name of the variant itself.
Regarding the name of the variant, it is indeed fine, as Revision is using the ordinal. But I guess that's the same for tuples.

Reply:
This change of removing in the middle of a enum will be supported better by the newer version of revision so I would wait with merging this PR until we get the new version of revision in. Otherwise we will need to add some extra annotations to this enum.

Apart from that this should work fine. The name of a type has no impact on how it's serialized, only the type and order of it's fields/variants.